### PR TITLE
Add nvt-ts touchscreen support for xiaomi-vince

### PR DESCRIPTION
**Pull request by barni2000**
There are some xiaomi-vince variant which comes with nvt touchscreen, this PR add support to it.

### CODE REVIEW

File: arch/arm64/boot/dts/qcom/msm8953-xiaomi-vince.dts [hunk 1]
```diff
@@ -137,6 +137,24 @@
...
+		reg = <0x01>;
+
+		interrupts-extended = <&tlmm 65 IRQ_TYPE_EDGE_FALLING>;
+		reset-gpio = <&tlmm 64 GPIO_ACTIVE_LOW>;
+		iovcc-supply = <&pm8953_l6>;
+
+		pinctrl-0 = <&ts_reset_active &ts_int_active>;
+		pinctrl-1 = <&ts_reset_suspend &ts_int_suspend>;
+		pinctrl-names = "default", "sleep";
+
```
Comment (z3ntu) [inline arch/arm64/boot/dts/qcom/msm8953-xiaomi-vince.dts:154]: remove this newline

**Review by z3ntu — APPROVED** [review]
otherwise looks fine

File: arch/arm64/boot/dts/qcom/msm8953-xiaomi-vince.dts [hunk 2]
```diff
@@ -137,6 +137,24 @@
 	status = "okay";
 };
 
+&i2c_3 {
+	status = "okay";
+
+	touchscreen@1 {
```
Comment (M0Rf30) [inline arch/arm64/boot/dts/qcom/msm8953-xiaomi-vince.dts:143]: downstream nvt node is not on @1
https://github.com/xiaomi-msm8953-devs/android_kernel_xiaomi_msm8953/blob/1ea7e46cc1d6635bb716a3effffb949bcacb11ad/arch/arm64/boot/dts/qcom/xiaomi/vince/vince-qrd.dtsi#L201C3-L201C13

`touchscreen@62` should be the right reg

**Review by M0Rf30 — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/qcom/msm8953-xiaomi-vince.dts (hunk 2 above)
Comment (z3ntu) [reply arch/arm64/boot/dts/qcom/msm8953-xiaomi-vince.dts:143]: Can't imagine touchscreen driver should work though if `reg` is wrong? Unless the driver is weirdly hardcoding the i2c address

Comment (barni2000) [reply arch/arm64/boot/dts/qcom/msm8953-xiaomi-vince.dts:143]: > Can't imagine touchscreen driver should work though if `reg` is wrong? Unless the driver is weirdly hardcoding the i2c address

I think it does something like this it is not working with 0x62, but the real i2c address is 0x62.

**Comment by barni2000** [comment]
Close in favor of #158

**Pull request closed without merging**